.github/workflows/deploy_thru_prod.yml

```diff
@@ -64,10 +70,11 @@ jobs:
     uses: ./.github/workflows/deploy_environment.yml
     with:
       environment: stage
+      oidc-domain: ${{ inputs.oidc-domain }}
       terraform-workspace: ${{ inputs.terraform-workspace || 'stage'}}
-      commit-identifier: ${{ inputs.commit-identifier }}
+      commit-identifier: ${{ inputs.commit-identifier || github.sha }}
       release-tag: ${{ needs.Set-Release-Tag.outputs.release-tag }}
-      terraform-root: terraform/stage
+      terraform-root: 'terraform/stage'
```

Comment:
are these quotes being added by a linter?

Reply:
The quotes around environment etc? No. I think it was from some code I copied and didn't remove it because it didn't break anything. If there is a preference to remove it, I can go ahead and remove.